Gemfile

```diff
@@ -5,9 +5,9 @@ source 'https://rubygems.org'
 # Specify your gem's dependencies in attr_default.gemspec
 gemspec
 
-gem 'actionmailer',  '>= 4.2.11.1'
-gem 'actionpack',    '>= 4.2.11.1'
-gem 'activesupport', '>= 4.2.11.1'
+gem 'actionmailer'
```

Comment:
Do you need these at all? They are in the gemspec already.

Reply:
Good point. I was able to remove those plus a lot of other junk. And I removed the groups since they weren't being used. https://github.com/Invoca/exception_handling/pull/64/commits/1e60f5cb40f69139673fa2a805c01a1217d57a3d